x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/categorization_view/examples_valid_callout.tsx

```diff
@@ -96,3 +118,28 @@ const AnalyzerUsed: FC<{ categorizationAnalyzer: CategorizationAnalyzer }> = ({
     </>
   );
 };
+
+const AllValidationChecks: FC<{ validationChecks: FieldExampleCheck[] }> = ({
+  validationChecks,
+}) => {
+  const list: EuiListGroupItemProps[] = Object.keys(VALIDATION_CHECK_DESCRIPTION).map((k, i) => {
```

Comment:
Looks like we may need an array to define the display order of the messages. Maybe in this order from `enum VALIDATION_RESULT` ?

```
  INSUFFICIENT_PRIVILEGES,
  NO_EXAMPLES,
  FAILED_TO_TOKENIZE,
  TOO_MANY_TOKENS,
  TOKEN_COUNT,
  MEDIAN_LINE_LENGTH,
  NULL_VALUES,
```

![image](https://user-images.githubusercontent.com/7405507/77063181-294bb780-69d5-11ea-84b7-c59b418a2f12.png)

Reply:
reordered in c69e101f6bbdaf2b834ed88169356f4c48a9ed44